### DISCUSSION TITLE
2 Dock UI features: Expand panels horizontally + Increase grid size

### DISCUSSION THREAD

**Discussion by hungLink** [godotengine/godot-proposals#10007] [Editor]
I created 2 improvement requests here:

https://github.com/godotengine/godot-proposals/issues/10005
https://github.com/godotengine/godot-proposals/issues/10006

Ultimately, I think that the dock in godot is great! I think it would be even better, and more customizable with these 2 features implemented.

What do you think?

**Comment by AThousandShips** [comment]
Since there's already proposals open, and people can discuss there, what's the purpose of this discussion?
